tests/test-update-direct.c

```diff
@@ -235,9 +237,11 @@ test_cancel_update (EosUpdaterFixture *fixture,
                                   &local_error);
   g_assert_no_error (local_error);
 
-  g_hash_table_insert (subserver->ref_to_commit,
+  g_hash_table_insert (leaf_commit_nodes,
```

Comment:
This can either be called `ref_to_commit` or `leaf_commit_nodes`. We should probably rename either `eos_test_subserver_ref_to_commit_new` or the `*_leaf_nodes` functions to be consistent here. Personally I prefer "leaf commit nodes" since it makes it clear that the caller needs to populate the graph elsewhere...